server/vulnerabilities/cpe_test.go

```diff
@@ -58,25 +56,14 @@ func TestSyncCPEDatabase(t *testing.T) {
 	nettest.Run(t)
 
 	client := fleethttp.NewClient()
-	// Disabling vcr because the resulting file exceeds the 100mb limit for github
-	r, err := recorder.NewAsMode("fixtures/nvd-cpe-release", recorder.ModeDisabled, client.Transport)
-	require.NoError(t, err)
-	defer r.Stop()
-
-	client.Transport = r
 
-	tempDir := os.TempDir()
-	dbPath := path.Join(tempDir, "cpe.sqlite")
-
-	err = os.Remove(dbPath)
-	if !errors.Is(err, os.ErrNotExist) {
-		require.NoError(t, err)
-	}
+	tempDir := t.TempDir()
 
 	// first time, db doesn't exist, so it downloads
-	err = SyncCPEDatabase(client, dbPath)
+	err := DownloadCPEDatabase(tempDir, client)
 	require.NoError(t, err)
 
+	dbPath := path.Join(tempDir, "cpe.sqlite")
```

Comment:
Should be `filepath.Join(..)`

Reply:
Nice catch. I also went through and fixed up all other occurrences in fleet.